docs/insomnia/git-sync.md

```diff
@@ -25,22 +27,33 @@ When configuring a remote repository, you will be prompted for the following inf
 * **Author Name**: The Git author name to store with each commit.
 * **Author Email**: The Git author email to store with each commit.
 * **Username**: The Git author username to match with the authentication token.
-* **Authentication Token**: The token needed to authenticate with remote repository provider, such as GitHub or BitBucket. If you have two-factor authentication (2FA) enabled on your account, it is unlikely you will be able to use your username and password. Instead, generate a Personal Access Token or App Password (see list below for links to documentation for your Git system).
+* **Authentication Token**: The token needed to authenticate with remote repository provider, such as GitHub or BitBucket. If you have two-factor authentication (2FA) enabled on your account, it is unlikely you will be able to use your username and password. Instead, generate a personal access token or app password (see list below for links to documentation for your Git system).
+
+{:.alert .alert-primary}
+**Note**: You may still fail to set up Git Sync properly due to not enough or the wrong types of Git permissions.
 
-Find instructions on how to create a Personal Access Token or App Password on the following platforms:
+Find instructions on how to create a personal access token or app password on the following platforms:
 
 * [Github](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+  * To push to and pull from a _public_ GitHub repo, scope at least `public_repo` when creating your personal access token.
```

Comment:
- It's worth documenting what's needed for private GitHub repositories. Many enterprise customers will have private repositories, and from my testing they need the entire `repo` scope.
- At least for GitHub, we can use a one-click link which has everything pre-filled:
  - public: `https://github.com/settings/tokens/new?description=insomnia-git-sync&scopes=public_repo`
  - private: `https://github.com/settings/tokens/new?description=insomnia-git-sync&scopes=repo`